src/commands/test/run/uitest.ts

```diff
@@ -116,6 +118,15 @@ export default class RunUITestsCommand extends RunTestsCommand {
     return preparer.prepare();
   }
 
+  protected getParametersFromOptions() : {} {
```

Comment:
I think that on the prepare - the necessary `chunker` stuff will be handled by shelling out to the `test-cloud.exe` - but wanted to double check that this was the case.